scripts/ci/test.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 ./scripts/ci/word-blacklist.sh
 
-yarn commitlint-travis
+yarn commitlint-azure-pipelines
```

Comment:
we probably need two ci test scripts then, one for travis and one for azure, until we remove travis